main.tf

```diff
@@ -1,7 +1,8 @@
 provider "triton" {
-    url = "https://${var.dc_name}.api.joyent.com"
-    account = "${var.triton_account}"
-    key_id = "${var.triton_key_id}"
+    # Commenting these out because they will take the SDC_URL, SDC_ACCOUNT, and SDC_KEY_ID env vars as defaults
```

Comment:
There are reasons for and against this, but using env vars is the surest way I know to keep credentials out of the repo.